src/main/java/it/gov/pagopa/arc/service/bizevents/BizEventsServiceImpl.java

```diff
@@ -18,15 +23,21 @@ public class BizEventsServiceImpl implements BizEventsService{
     private final BizEventsConnector bizEventsConnector;
     private final BizEventsTransactionDTO2TransactionDTO transactionDTOMapper;
     private final BizEventsTransactionsListDTO2TransactionsListDTO transactionsListDTOMapper;
+    private final BizEventsCartItem2CartItemDTO cartItemDTOMapper;
+    private final BizEventsTransactionDetails2TransactionDetailsDTO transactionDetailsDTOMapper;
 
     public BizEventsServiceImpl(@Value("${rest-client.biz-events.fake-fiscal-code}") String fakeFiscalCode,
                                 BizEventsConnector bizEventsConnector,
                                 BizEventsTransactionDTO2TransactionDTO transactionDTOMapper,
-                                BizEventsTransactionsListDTO2TransactionsListDTO transactionsListDTOMapper) {
+                                BizEventsTransactionsListDTO2TransactionsListDTO transactionsListDTOMapper,
+                                BizEventsCartItem2CartItemDTO cartItemDTOMapper,
+                                BizEventsTransactionDetails2TransactionDetailsDTO transactionDetailsDTOMapper) {
         this.fakeFiscalCode = fakeFiscalCode;
         this.bizEventsConnector = bizEventsConnector;
         this.transactionDTOMapper = transactionDTOMapper;
         this.transactionsListDTOMapper = transactionsListDTOMapper;
+        this.cartItemDTOMapper = cartItemDTOMapper;
+        this.transactionDetailsDTOMapper = transactionDetailsDTOMapper;
     }
```

Comment:
all this boilerplate code can be replaced by lombok `RequiredArgsConstructor` annotation